java/system-test/system-test-configuration/src/main/java/sleeper/systemtest/configuration/SystemTestProperty.java

```diff
@@ -30,79 +30,107 @@
 public interface SystemTestProperty extends InstanceProperty {
-    SystemTestProperty NUMBER_OF_WRITERS = Index.propertyBuilder("sleeper.systemtest.writers")
-            .description("The number of containers that write random data")
-            .validationPredicate(Objects::nonNull).build();
-    SystemTestProperty NUMBER_OF_RECORDS_PER_WRITER = Index.propertyBuilder("sleeper.systemtest.records-per-writer")
-            .description("The number of random records that each container should write")
-            .validationPredicate(Objects::nonNull).build();
+    SystemTestProperty SYSTEM_TEST_ID = Index.propertyBuilder("sleeper.systemtest.standalone.id")
+            .description("The id of the deployment, when deploying standalone.")
+            .editable(false).build();
+    SystemTestProperty SYSTEM_TEST_ACCOUNT = Index.propertyBuilder("sleeper.systemtest.standalone.account")
+            .description("The AWS account when deploying standalone.")
+            .editable(false).build();
+    SystemTestProperty SYSTEM_TEST_REGION = Index.propertyBuilder("sleeper.systemtest.standalone.region")
+            .description("The AWS region when deploying standalone.")
+            .editable(false).build();
+    SystemTestProperty SYSTEM_TEST_VPC_ID = Index.propertyBuilder("sleeper.systemtest.standalone.vpc")
+            .description("The id of the VPC to deploy to, when deploying standalone.")
+            .editable(false).build();
+    SystemTestProperty SYSTEM_TEST_JARS_BUCKET = Index.propertyBuilder("sleeper.systemtest.standalone.jars.bucket")
+            .description("The S3 bucket containing the jar files of the Sleeper components, when deploying standalone.")
+            .runCDKDeployWhenChanged(true).build();
+    SystemTestProperty SYSTEM_TEST_REPO = Index.propertyBuilder("sleeper.systemtest.repo")
+            .description("The image in ECR used for writing random data to the system")
+            .validationPredicate(Objects::nonNull)
+            .runCDKDeployWhenChanged(true).build();
+    SystemTestProperty SYSTEM_TEST_CLUSTER_ENABLED = Index.propertyBuilder("sleeper.systemtest.cluster.enabled")
+            .description("Whether to deploy the system test cluster for data generation")
+            .defaultValue("true").validationPredicate(Utils::isTrueOrFalse)
+            .runCDKDeployWhenChanged(true).build();
+    SystemTestProperty SYSTEM_TEST_CLUSTER_NAME = Index.propertyBuilder("sleeper.systemtest.cluster")
+            .description("The name of the ECS cluster where system test tasks will run")
+            .systemDefined(true).build();
+    SystemTestProperty SYSTEM_TEST_BUCKET_NAME = Index.propertyBuilder("sleeper.systemtest.bucket")
+            .description("The name of the bucket where system test data will be stored")
+            .systemDefined(true).build();
+    SystemTestProperty WRITE_DATA_TASK_DEFINITION_FAMILY = Index.propertyBuilder("sleeper.systemtest.task-definition")
```

Comment:
It's a bit odd to have a dash in the property name, i think we normally either concatenate the words or separate them with a dot.

Edit: I realise this was just moved from below, but i thought i'd mention it as it looked a bit strange.

Reply:
I've changed it.